Caboodle/DeviceInfo/DeviceInfo.android.cs

```diff
@@ -0,0 +1,248 @@
+﻿using System;
+using System.Globalization;
+using Android.App;
+using Android.Content;
+using Android.Content.PM;
+using Android.Content.Res;
+using Android.OS;
+using Android.Provider;
+using Android.Runtime;
+using Android.Views;
+using CaboodlePlatform = Microsoft.Caboodle.Platform;
+
+namespace Microsoft.Caboodle
+{
+    public static partial class DeviceInfo
+    {
+        const int tabletCrossover = 600;
+
+        static string GetIdentifier()
+        {
+            var id = GetSerial();
+            if (string.IsNullOrWhiteSpace(id) || id == Build.Unknown || id == "0")
+            {
+                try
+                {
+                    id = GetSecureSetting(Settings.Secure.AndroidId);
+                }
+                catch
+                {
+                }
+            }
+            return id;
+        }
+
+        static string GetModel() => Build.Model;
+
+        static string GetManufacturer() => Build.Manufacturer;
+
+        static string GetDeviceName()
+        {
+            var name = GetSystemSetting("device_name");
+            if (string.IsNullOrWhiteSpace(name))
+                name = Model;
+            return name;
+        }
+
+        static string GetVersionString() => Build.VERSION.Release;
+
+        static string GetAppPackageName() => CaboodlePlatform.CurrentContext.PackageName;
+
+        static string GetAppName()
+        {
+            var applicationInfo = CaboodlePlatform.CurrentContext.ApplicationInfo;
+            var packageManager = CaboodlePlatform.CurrentContext.PackageManager;
+            return applicationInfo.LoadLabel(packageManager);
+        }
+
+        static string GetAppVersionString()
+        {
+            SetAppVersions();
+            return appVersionString;
+        }
+
+        static string GetAppBuild()
+        {
+            SetAppVersions();
+            return appBuild;
+        }
+
+        static string GetPlatform() => Platforms.Android;
```

Comment:
Why is this a method? Can't it just be Platform?

Reply:
It is a property, in the shared file. This is just the private implementation.